cadence/nft-lock-box/NFTLockBox.cdc

```diff
@@ -0,0 +1,168 @@
+import NonFungibleToken from {{{ imports.NonFungibleToken }}}
+import FungibleToken from {{{ imports.FungibleToken }}}
+
+pub contract NFTLockBox {
+
+    // The Claimed event is emitted when an NFT
+    // is successfully claimed from a lock box.
+    pub event Claimed(
```

Comment:
I am not sure about the event name as it doesn't resonate with the LockBox meaning,  Maybe a better name would be `Retrieved`

Reply:
Hmm, to me "claim from the lockbox" feels more natural than "retrieve from the lockbox"  🤔 

I've also been calling the private keys "claim keys", which has felt logical to me.